### ISSUE TITLE
Emoji like on github

### ISSUE THREAD

**Issue by vetash** [RocketChat/feature-requests#15]
How i can change these flat emoji on emoji like in slack?

**Comment by amoebav** [comment]
Yes, I would like to change the entire emoji set. I don't want to use the emoji set provide by JoyPixels.

**Comment by gBasil** [comment]
This issue was made 4 years ago and still doesn't have a resolution. Have there been any developments?

**Comment by ThatOneCalculator** [comment]
+1... JoyPixels is now very outdated, Twemoji is still maintained but why not just allow the system default emoji?
